client/packages/requisitions/src/ResponseRequisition/ListView/ListView.tsx

```diff
@@ -51,6 +51,10 @@ export const ResponseRequisitionListView: FC = () => {
         key: 'status',
         condition: 'equalTo',
       },
+      {
+        key: 'shipmentCreated',
```

Comment:
Mark mentioned 'a shipment has been created' filter, is that name too big to fit in filters ?

Reply:
How about `has shipment created`?

Reply:
I would go with what product owner has decided in the issue comment.